### ISSUE TITLE
Documentation should be updated to state what a "first class" environment variable is

### ISSUE THREAD

**Issue by silbinarywolf** [silverstripe/silverstripe-framework#8204]
**The problem**
It's unclear on what is meant here (had a client literally ask us and pointed to the documentation)

**The solution**
Add parenthesis with *something* like "Configure at OS level".
I have a feeling there is more to cover here that I'm not across, but multiple examples of "first class" env vars would help!

**Suggested copy change:**
>Sensitive credentials should not be stored in a VCS or project code and should only be stored on the environment in question. When using live environments the use of .env files is discouraged and instead one should use environment variables set at the OS level.

**Source:**
https://docs.silverstripe.org/en/4/getting_started/environment_management/#security-considerations

**Comment by maxime-rainville** [comment]
I agree that "first class" variables sounds a bit cryptic. Not sure "set at the OS level" is an accurate description however. Normally those variables should be set in your web server configuration.

That's [how Symfony describes it](http://symfony.com/doc/current/configuration/external_parameters.html#configuring-environment-variables-in-production). Any other views or suggestion?

**Comment by silbinarywolf** [comment]
How about this then?
>Sensitive credentials should not be stored in a VCS or project code and should only be stored on the environment in question. When using live environments the use of .env files is discouraged and it is recommended to configure these at the web server level, ie. Apache or Nginx configurations

**Comment by NightJar** [comment]
How about a link to some relevant info, say like https://en.wikipedia.org/wiki/Environment_variable

Bad example because:
https://motherboard.vice.com/en_us/article/ne7xzq/wikipedias-science-articles-are-elitist
https://scholarlykitchen.sspnet.org/2012/09/24/wikipedias-writing-tests-show-its-too-sophisticated-for-its-audience/

But you get the idea :)
